File: 217 Contains Duplicate/Java/Solution.java
import java.util.HashSet;

public class Solution {
    public static boolean containsDuplicate(int[] nums) {
        HashSet<Integer> set = new HashSet<Integer>();

        for (int num : nums) {
            if (!set.add(num))
                return true;
        }

        return false;
    }
}

File: 217 Contains Duplicate/JavaScript/solution.js
/**
 * @param {number[]} nums
 * @return {boolean} 
 */
const containsDuplicate = (nums) => {
    const numSet = new Set();
    let isDuplicate = false;

    nums.forEach(num => {
      let oldSize = numSet.size;
      numSet.add(num);
      if (numSet.size === oldSize)
        isDuplicate = true;
    });
  
    return isDuplicate;
  }

module.exports = {
    containsDuplicate
}